plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/FilesTable.java

```diff
@@ -15,34 +15,42 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import io.airlift.slice.Slices;
```

Comment:
`Avoid materializing all results in Iceberg system table at once` -> `Avoid materializing all table files in Iceberg $files system table at once`

Reply:
somehow GitHub commit did not recognize `$files`. I have updated it now.